src/aiai_eval/task_configs.py

```diff
@@ -17,6 +17,9 @@ def get_all_dataset_tasks() -> Dict[str, DatasetTask]:
 
 NER = DatasetTask(
     name="ner",
+    dataset_name="dane",
+    pretty_dataset_name="the truncated version of DaNE",
+    huggingface_id="ScandEval/dane-mini",
```

Comment:
These are ScandEval-only datasets. We should use the official DaNE dataset here (https://huggingface.co/datasets/dane)